File: 151001/Matsiushenko/lab1/src/main/java/by/bsuir/vladislavmatsiushenko/impl/dto/UserResponseTo.java
package by.bsuir.vladislavmatsiushenko.impl.dto;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class UserResponseTo {
    private long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151003/Barylko/SQL/TaskSQL/Dto/Response/UserResponseTo.cs
using System.Text.Json.Serialization;

namespace TaskREST.Dto.Response;

public record UserResponseTo(
    [property: JsonPropertyName("id")] long id,
    [property: JsonPropertyName("login")] string Login,
    [property: JsonPropertyName("password")]
    string Password,
    [property: JsonPropertyName("firstname")]
    string FirstName,
    [property: JsonPropertyName("lastname")]
    string LastName
);